src/main/java/taxi/controller/car/AddDriverToCarController.java

```diff
@@ -20,16 +21,24 @@ public class AddDriverToCarController extends HttpServlet {
     @Override
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        HttpSession session = req.getSession();
+        if (session.getAttribute("driver_id") != null) {
+            Long driverId = Long.parseLong(String.valueOf(session.getAttribute("driver_id")));
+            req.setAttribute("driver_name", driverService.get(driverId).getName());
+        }
```

Comment:
you have this logic in several places, remove it everywhere, you can add to a session such attribute and take this value from the session in every controller where you need it and you don't need to check for null of driver_id, you know exactly that you can have it in LoginController, so set driver_name to session there one time